QGCApplication.pro

```diff
@@ -78,6 +81,11 @@ QT += testlib
 # OS Specific settings
 #
 
+AndroidBuild {
+    DEFINES += __android__
+    DEFINES += __STDC_LIMIT_MACROS
```

Comment:
The compiler settings are all in QGCCommon.pri so that they are common to all projects. You'll find a __STC_LIMIT_MACROS section already in there.